apps/mobile/src/components/Badge/Badge.styles.js

```diff
@@ -1,5 +1,5 @@
 import { StyleSheet } from 'react-native'
-import { persimmon } from 'style/colors'
+import { persimmon } from '@hylo/presenters/colors'
```

Comment:
Oh, why move the colors into shared??

Reply:
yeah, in fact i have no idea how it was working before! the PostPresenter was importing style/colors which lives in the mobile app 🤔🤔🤔